### DISCUSSION TITLE
Fluent translations

### DISCUSSION THREAD

**Discussion by hydroper** [bevyengine/bevy#9147] [Ideas]
There could be an optional feature for quickly using Fluent Translation Lists in a Bevy project. I'm not familiar with other ways of internationalization though: I've no pratical experience with translations either. If anyone is interested, some months ago I built a crate called `rialight::intl` which comes with a `Ftl` object.

- Here's more information: https://github.com/rialight/docs/blob/main/internationalization/basics.md
- Here's the source: https://github.com/rialight/api/tree/master/rialight_intl (`rialight_intl`)

It supports loading FTL from file system and HTTP (uses some crate called `reqwest`). I'd like to do a few internal improvements, such as how text direction can be derived from a locale from the `icu::locid` crate (it was not done very reliably).

**Comment by alice-i-cecile** [comment]
Yep, internationalization via `fluent` is on my wishlist. There appears to be a crate for it already even! https://github.com/kgv/bevy_fluent

Once these are more tested and vetted I'd be happy for Bevy to endorse or upstream an implementation layer.

**Comment by TimJentzsch** [comment]
There is also https://github.com/TimJentzsch/bevy_mod_localization, but it's still under development.
